src/main/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTask.java

```diff
@@ -152,6 +157,14 @@ public void setMaxWarnings(int maxWarnings) {
     }
 
     /**
+     * Adds a source path.
+     * @param sourcePath the source path to add.
+     */
+    public void addSourcePath(Path sourcePath) {
```

Comment:
@soon , please keep commenting on each point as "done" for self-recheck and hint to me that you addressed a point.